### PR TITLE
Add Lambda token to frontier

### PR DESCRIPTION
**Pull request by steveliupai**
@JeremyParish69 Hi, Jeremy. I submit an new PR to add lambda to both frontier and frontend. Pls review and merge, thank you so much!

### CODE REVIEW

**Comment by steveliupai** [comment]
@JeremyParish69 If use prettier for formatting json file is the problem, i can remove those fixes.

**Comment by JeremyParish69** [comment]
@steveliupai Lambda should first be added to the Osmosis frontend repo here: https://github.com/osmosis-labs/osmosis-frontend/tree/master/packages/web/config (and it will only show on the Frontier site as long as it's not marked as 'verified'). And after that it can then be added here to the assetlists repo, but it should only be added to the Frontier file (osmosis-frontier.assetlist.json); it should not be added to osmosis-1.assetlist.json file.

**Comment by steveliupai** [comment]
> https://github.com/osmosis-labs/osmosis-frontend/tree/master/packages/web/config

@JeremyParish69 I followed these steps marked in [docs](https://docs.osmosis.zone/overview/integrate/token-listings). I didn't know i should submit PR to frontend repo first.

**Comment by steveliupai** [comment]
> @steveliupai Lambda should first be added to the Osmosis frontend repo here: https://github.com/osmosis-labs/osmosis-frontend/tree/master/packages/web/config (and it will only show on the Frontier site as long as it's not marked as 'verified'). And after that it can then be added here to the assetlists repo, but it should only be added to the Frontier file (osmosis-frontier.assetlist.json); it should not be added to osmosis-1.assetlist.json file.

Does this mean i should create pool before register to assetlists?

**Comment by JeremyParish69** [comment]
> > https://github.com/osmosis-labs/osmosis-frontend/tree/master/packages/web/config
> 
> @JeremyParish69 I followed these steps marked in [docs](https://docs.osmosis.zone/overview/integrate/token-listings). I didn't know i should submit PR to frontend repo first.

Apologies for the confusion. These docs are overdue for an update. You can leave this PR open for now, as it will be needed, but please add the token to the frontend repo first. A pool isn't required beforehand, but is recommended to have ASAP.

**Comment by steveliupai** [comment]
> > > https://github.com/osmosis-labs/osmosis-frontend/tree/master/packages/web/config
> > 
> > 
> > @JeremyParish69 I followed these steps marked in [docs](https://docs.osmosis.zone/overview/integrate/token-listings). I didn't know i should submit PR to frontend repo first.
> 
> Apologies for the confusion. These docs are overdue for an update. You can leave this PR open for now, as it will be needed, but please add the token to the frontend repo first. A pool isn't required beforehand, but is recommended to have ASAP.

Thank you so much for the informations! I will make an PR to frontend repo first.

**Comment by steveliupai** [comment]
> > > https://github.com/osmosis-labs/osmosis-frontend/tree/master/packages/web/config
> > 
> > 
> > @JeremyParish69 I followed these steps marked in [docs](https://docs.osmosis.zone/overview/integrate/token-listings). I didn't know i should submit PR to frontend repo first.
> 
> Apologies for the confusion. These docs are overdue for an update. You can leave this PR open for now, as it will be needed, but please add the token to the frontend repo first. A pool isn't required beforehand, but is recommended to have ASAP.

@JeremyParish69  Hi Jeremy, I just made an [PR](https://github.com/osmosis-labs/osmosis-frontend/pull/907) to frontend

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -3244,6 +3219,33 @@
...
+      "symbol": "LAMB",
+      "ibc": {
+        "source_channel": "channel-2",
+        "dst_channel": "channel-378",
+        "source_denom": "ulamb"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lambda/images/lambda.png"
+      },
+      "coingecko_id": "lambda"
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:3248]: It looks likke on CoinGecko this points to the wrong lambda project. Can you confirm?

Comment (steveliupai) [reply osmosis-1/osmosis-frontier.assetlist.json:3248]: Its the same project, we launched our mainnet and we deploy a bridge to bridge ERC20 Lamb to mainnet.

Here is our bridge frontend   https://app.lambda.im/crosschain/ethereum_bridge

Comment (steveliupai) [reply osmosis-1/osmosis-frontier.assetlist.json:3276]: Btw, i would like to know how to get marked as `verified`

**Pull request merged**